product_docs/docs/pgd/5/durability/lag-control.mdx

```diff
@@ -4,46 +4,76 @@ redirects:
-PGD group nodes complies with these organizational objectives. It does so
-by controlling the extent of replication lag between PGD nodes.
-
-Some of these objectives include the following:
-
--  Recovery point objective (RPO) specifies the maximum-tolerated
-   amount of data that can be lost due to unplanned events, usually
-   expressed as an amount of time. In non-replicated systems, RPO
-   is used to set backup intervals to limit the risk of lost committed
-   data due to failure. For replicated systems, RPO determines the
-   acceptable amount of committed data that hasn't been safely applied
-   to one or more peer nodes.
-
--  Resource constraint objective (RCO) acknowledges that there are finite
-   storage constraints. This storage includes database files, WAL, and
-   temporary or intermediate files needed for continued operation.
-   For replicated systems, as lag increases the demands on these storage
-   resources also increase.
-
--  Group elasticity objective (GEO) ensures that any node isn't
-   originating new data at a clip that can't be acceptably saved to
-   its peer nodes. When that's the case, then the detection of that
-   condition can be used as one metric in the decision to expand the
-   number of database nodes. Similarly, when that condition abates then
-   it might influence the decision to contract the number of database nodes.
-
-Lag Control manages replication lag by controlling the rate at which client
-connections can commit READ WRITE transactions. Replication lag is
-measured either as lag time or lag size, depending on the objectives
-to meet. Transaction commit rate is regulated using a configured
-PGD commit-delay time.
+Commit scope kind: `LAG CONTROL`
+
+## Overview
+
+Lag Control provides a mechanism where, if replication is running outside of
+limits set, a delay is injected into the origin node's client connections after
+processing transactions that make replicable updates. This delay is designed to
+slow the incoming transactions and bring replication back within the defined
+limits.
+
+## Background
+
+The data throughput of database applications on a PGD origin node can exceed the
+rate at which committed data can replicate to downstream peer nodes.
+
+If this imbalance persists, it can put satisfying organizational objectives 
+such as RPO, RCO, and GEO at risk.
+
+!!! information
+
+-  RPO - Recovery point objective specifies the maximum-tolerated amount of data
+   that can be lost due to unplanned events, usually expressed as an amount of
+   time. In PGD, RPO determines the acceptable amount of committed data that
+   hasn't been applied to one or more peer nodes.
+
+-  RCO - Resource constraint objective acknowledges that there is finite storage
+   available. In PGD, as lag increases the demands on these storage resources
+   also increase.
+
+-  GEO - Group elasticity objective ensures that any node isn't originating new
+   data at a rate that can't be saved to its peer nodes. 
+
+!!!
```

Comment:
This part renders badly: https://deploy-preview-4677--edb-docs-staging.netlify.app/docs/pgd/latest/durability/lag-control/#background

Reply:
Fixed - though we need to track down the broken !!!info admonition.